src/sentry/replaystore/migrations/0001_init.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.24 on 2022-04-21 00:43
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
```

Comment:
we don't want this to run in production, do we just change `is_dangerous = True`?

Reply:
That will work. But this will run in self-hosted and ST regardless